packages/app/src/components/inspector/ColorPickerGradient.tsx

```diff
@@ -73,15 +52,19 @@ const SwatchesList = memo(function SwatchesList({
           gradientType,
           180,
         );
+        const handleContextMenu = () => {
```

Comment:
If I don't send the event?: React.MouseEvent, to this onContextMenu it doesn't trigger for the list. I need it to look like this so I can set which gradient I am currently selecting

Reply:
Our `ListView.Row` already has a built-in menu, so we should be able to use `menuItems` and `onSelectMenuItem` to get a context menu. There might still be problems afterward, but lets try that first